nin/backend/fontgen.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs-promise');
+const glob = require('glob');
+const path = require('path');
+
+
+async function fontGen(pathPrefix) {
+
+  let resolver;
+  let promise = new Promise(resolve => {
```

Comment:
Is this code equivalent to wrapping the glob inside the `new Promise(resolve => {
}` code? Can there be a race condition  now that the scheduler runs the glob callback before the callback given to the promise, so resolver is undefined?